java/pom.xml

```diff
@@ -657,6 +657,7 @@
     <module>performance</module>
     <module>algorithm</module>
     <module>adapter/avro</module>
+    <module>gandiva</module>
```

Comment:
I think you should be able to do this by using the arrow-jni profile defined below?   Is there something that doesn't work when using it?

Reply:
Note, this isn't included by default because it requires JNI modules

Reply:
I see.

Let me go back to IDE to check arrow-jni profile

Reply:
Did using arrow-jni work for you?

Reply:
Nope. I didn't make arrow-jni work. Maybe it is just because I am not familiar with my IDE.

Anyway, I will close this PR if there is no other question. I will just adding module locally to make sure my IDE works.